### PR TITLE
Update ci.yml and add .golangci.yml

### PR DESCRIPTION
**Pull request by Smolevich**
- Add .golanci.yml
- Refactor ci.yml

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -44,14 +46,10 @@ jobs:
...
     - name: run linters
       run: |
-        $GITHUB_WORKSPACE/golangci-lint run --out-format=tab --disable-all --tests=false --enable=unconvert \
-          --enable=megacheck --enable=structcheck --enable=gas --enable=gocyclo --enable=dupl --enable=misspell \
-          --enable=unparam --enable=varcheck --enable=deadcode --enable=typecheck \
-          --enable=ineffassign --enable=varcheck ./... ;
+        $GITHUB_WORKSPACE/golangci-lint run --config $GITHUB_WORKSPACE/.golangci.yml ./... ;
 
     - name: submit coverage
       run: |
```
Comment (paskal) [inline .github/workflows/ci.yml]: Shouldn't be multiline I guess.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -44,14 +46,10 @@ jobs:
 
     - name: run linters
       run: |
```
Comment (paskal) [inline .github/workflows/ci.yml]: Shouldn't be multiline anymore?

File: .golangci.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    - gocyclo
+    - dupl
+    - misspell
+    - unparam
+    - varcheck
+    - deadcode
+    - typecheck
+    - ineffassign
+    - varcheck
+  disable-all: false
```
Comment (paskal) [inline .golangci.yml]: `true` in the original configuration.

**Comment by paskal** [comment]
Thanks a lot! I forgot to ask you to update dockerfile as well, would you be so kind as to prepare PR for that?

**Comment by Smolevich** [comment]
> Thanks a lot! I forgot to ask you to update dockerfile as well, would you be so kind as to prepare PR for that?

yes, i prepare PR for this case

**Pull request merged**